lit_gpt/speed_monitor.py

```diff
@@ -293,10 +294,28 @@ def eval_end(self, eval_elapsed: float):
         self.total_eval_wct += eval_elapsed  # seconds
 
 
+def plugin_to_compute_dtype(plugin: Precision) -> torch.dtype:
```

Comment:
This could be in Fabric/PL

Reply:
Maybe something like `plugin.compute_dtype` and `plugin.param_dtype`.